byzcoin/bcadmin/main.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/dedis/onet/log"
 	"github.com/dedis/onet/network"
 
+	"encoding/json"
+	qrgo "github.com/skip2/go-qrcode"
```

Comment:
CISC uses github.com/qantik/qrgo. Why use a different generator here? Or can we change CISC to use the same one? Or are they actually forks of the same code?

Anyway, please find a way to keep the number of dependencies the same.